--- a/pkgs/by-name/da/davinci-resolve/package.nix
+++ b/pkgs/by-name/da/davinci-resolve/package.nix
@@ -56,7 +56,7 @@
   davinci = (
     stdenv.mkDerivation rec {
       pname = "davinci-resolve${lib.optionalString studioVariant "-studio"}";
-      version = "21.0.1";
+      version = "21.0.2";
 
       nativeBuildInputs = [
         appimageTools.appimage-exec
@@ -78,7 +78,7 @@
             outputHashAlgo = "sha256";
             outputHash =
               if studioVariant then
-                "sha256-8JN3ptd8jcacxHihZHXuhdkyambUsnFIj+AruvpztKI="
+                "sha256-TrK8KqEyWkDQsgAzPp7kZIxjQRMOR5BxW0vPZky7sHg="
               else
                 "sha256-ioAqvqHjwFX1ec6fDoxg2VUZy1moYoGx/aEewDuN1+g=";
 
