diff --git a/src/search.ts b/src/search.ts
index 0d7a45f..f93775a 100644
--- a/src/search.ts
+++ b/src/search.ts
@@ -373,12 +373,8 @@ export class Search {
 
         this.list.show();
 
-        const vscroll = (this.scroller as any).get_vscroll_bar();
-        if ((this.scroller as any).vscrollbar_visible) {
-            vscroll.show();
-        } else {
-            vscroll.hide();
-        }
+        // Show vertical scrollbar only when needed, hide horizontal always
+        //this.scroller.set_policy(St.PolicyType.NEVER, St.PolicyType.AUTOMATIC);
 
         if (id === 0) {
             GLib.idle_add(GLib.PRIORITY_DEFAULT, () => {
